### PR TITLE
Add "django_example_ynh"

### PR DESCRIPTION
**Pull request by jedie**
https://github.com/YunoHost-Apps/django_example_ynh

Demo YunoHost Application to demonstrate the integration of a Django project under YunoHost.

### CODE REVIEW

File: apps.json [hunk 1]
```diff
@@ -668,14 +668,22 @@
...
+    },
     "django-fmd": {
         "category": "iot",
         "level": 7,
         "state": "working",
         "url": "https://github.com/YunoHost-Apps/django-fmd_ynh"
     },
     "django-for-runners": {
-        "state": "inprogress",
+        "state": "working",
```
Comment (tituspijean) [inline apps.json:686]:
```suggestion
        "category": "small_utilities",
        "state": "working",
```

**Review by tituspijean — CHANGES_REQUESTED** [review]

File: apps.json (hunk 1 above)
Comment (tituspijean) [reply apps.json:686]: Let's add a category to Django-for-Runners, otherwise LGTM!

Comment (jedie) [reply apps.json:686]: i searched around for a suitable category, but there is no one. "small_utilities" seems to be wrong ;) Maybe "office" ?

About [django-for-runners](https://github.com/jedie/django-for-runners): 

> Store your GPX tracks of your running (or other sports activity) in django.

Comment (tituspijean) [reply apps.json:686]: I agree it is not perfect, but that's where we put Owntracks and µLogger.

Comment (tituspijean) [reply apps.json:686]: Anyways, that's unrelated to the aim of this PR. I dunno why your commit for Runners ended up here.

Comment (jedie) [reply apps.json:686]: OK. I create a new PR: https://github.com/YunoHost/apps/pull/1497

**Comment by yalh76** [comment]
I'm not sure we want that.
Usually YunoHost package and the upstream application are into differents repositories

**Comment by jedie** [comment]
@yalh76 the story behind this is here: https://forum.yunohost.org/t/python-package-django-ynh-as-a-helper-to-package-django-based-projects/13769/4

tl;tr; It's needed to be a "official" package to pass the CI:

grafik

**Pull request merged**